CRM/Civicase/Hook/Permissions/ExportCasesAndReports.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use CRM_Civicase_Service_ExportPermission as ExportPermission;
+
+/**
+ * Adds cases and reports export permissions.
+ */
+class CRM_Civicase_Hook_Permissions_ExportCasesAndReports {
+
+  /**
+   * Civi permissions array.
+   *
+   * @var array
+   */
+  private $permissions;
+
+  /**
+   * Case Export Permission service.
+   *
+   * @var CRM_Civicase_Service_ExportPermission
+   */
+  private $exportPermissionService;
+
+  /**
+   * CRM_Civicase_Hook_Permissions_CaseCategory constructor.
+   *
+   * @param array $permissions
+   *   Civi permissions array.
+   */
+  public function __construct(array &$permissions) {
+    $this->permissions = &$permissions;
+    $this->exportPermissionService = new ExportPermission();
```

Comment:
I don't think we need this new service tbh. We can add the export action permission directly in this hook class. Since we are not doing anything dynamic or related to case type categories, it's fine if we add it directly.